### PR TITLE
Add support for showing and hiding nodes

### PR DESCRIPTION
**Pull request by dragly**
Also request updates for changing colors where this was missing.

### CODE REVIEW

**Comment by dragly** [comment]
This depends on #324. Wait with merging until that is merged.

**Comment by dragly** [comment]
Also dependent on #323, waiting for that to go through CI

**Comment by dragly** [comment]
Strike that, this is ready for review.

File: viewer/src/migration/Cognite3DModel.ts [hunk 1]
```diff
@@ -149,18 +155,43 @@ export class Cognite3DModel extends THREE.Object3D {
...
+  async hideNode(nodeId: number, makeGray?: boolean): Promise<void> {
+    if (makeGray) {
+      throw new NotSupportedInMigrationWrapperError();
+    }
+    try {
+      const treeIndex = await this.nodeIdAndTreeIndexMaps.getTreeIndex(nodeId);
+      this.hiddenNodes.add(treeIndex);
+      this.cadNode.requestNodeUpdate([treeIndex]);
+    } catch (error) {
+      console.error(`Cannot reset color of ${nodeId} because of error:`, error);
```
Comment (larsmoa) [inline viewer/src/migration/Cognite3DModel.ts]: Throw error to client.

**Comment by dragly** [comment]
@larsmoa adapted to your comments now

**Pull request merged**